### PR TITLE
Support for non-RStudio use

### PR DESCRIPTION
**Pull request by jonocarroll**
I've added support for terminal use (not requiring RStudio API). This moves rstudioapi to Suggests. 

Use in a terminal now either `cat()`s the output and/or pushes to the clipboard (if requested).

### CODE REVIEW

**Comment by jonocarroll** [comment]
There's more apparent changes than there really are - I seem to have auto-indented a few blocks.

**Comment by jonocarroll** [comment]
Tested on a remote rstudio-server; `tribble_paste(head(mtcars))` works fine and pastes right into the script.

Pastes fine in an `ssh` terminal to an R session too.

**Comment by MilesMcBain** [comment]
There's some nice improvements here, thanks Jono. One thing that is concerning me is that `tribble_paste()` is getting a bit monolithic and hard to follow. I was thinking it might be better to create two functions, one for the addin and one for clipboard/terminal use. We'd pull out the guts into a new function called by both, they just to the data acquisition and  output in their own way. What do you think?

**Comment by MilesMcBain** [comment]
I suppose it's just output really. But my sense is that if you are going to use the clipboard output feature, the other output is just in your way.

**Comment by MilesMcBain** [comment]
I've made a refactor which adds a bunch of _format functions for use with clipboard output. Needs doco and some more tests but getting close to ready for new release.

**Comment by MilesMcBain** [comment]
Implemented in version 2
